TLM/TLM/Manager/Impl/AdvancedParkingManager.cs

```diff
@@ -2687,19 +2686,19 @@ bool citizenDebug
                     }
 
                     Vector3 position = transformMatrix.MultiplyPoint(prop.m_position);
-                    if (CustomPassengerCarAI.FindParkingSpaceProp(
-                        isElectric,
-                        ignoreParked,
-                        propInfo,
-                        position,
-                        building.m_angle + prop.m_radAngle,
-                        prop.m_fixedHeight,
-                        refPos,
-                        vehicleInfo.m_generatedInfo.m_size.x,
-                        vehicleInfo.m_generatedInfo.m_size.z,
-                        ref propMinDistance,
-                        ref parkPos,
-                        ref parkRot))
+                    if (Constants.ManagerFactory.GameConnectionManager.PassengerCarAIConnection.FindParkingSpaceProp(
```

Comment:
adding argument names might improve readability on this long call
plus a const for `Constants.ManagerFactory.GameConnectionManager.PassengerCarAIConnection` might help readability a little too

Reply:
Yes, I don't like such long names, so in every Harmony patch I stored references to methods to improve readability and also it's easier to compare with vanilla code if necessary. I'll change that part too 😉